### DISCUSSION TITLE
Cancel Sales Invoice

### DISCUSSION THREAD

**Discussion by GienRose4** [lavaloon-eg/ksa_compliance#171] [General]
Hi good day. Is there a way to cancel and delete a submitted and accepted invoice?

**Comment by mhaggag** [comment]
Hello. ZATCA only models submitting invoices, debit notes and credit notes--there's no cancellation or deletion. Once documents are submitted to ZATCA, they're immutable. In this case, you can return the invoice with a credit note (essentially cancelling it out both on ERPNext and on ZATCA from an accounting standpoint) and then issue a new one with corrected information.

EDIT: Clarification.

**Reply by GienRose4** [reply]
Thank you @mhaggag
